wcivf/apps/api/views.py

```diff
@@ -74,6 +80,11 @@ def list(self, request, *args, **kwargs):
                 "ballot_locked": postelection.locked,
                 "replaced_by": postelection.replaced_by,
                 "candidates": candidates,
+                "voting_system": VotingSystemSerializer(
+                    postelection.voting_system
+                ).data,
+                "seats_contested": postelection.winner_count,
+                "organisation_type": postelection.post.organization_type,
```

Comment:
What are your thoughts on what to do with this field once the data gets to devs.DC? Are you thinking of just using it for sorting the ballots and then binning the value, or exposing it?

If we're just going to use it for sorting and discard it, I don't suppose this matters:

If we're going to also expose it via the `/postcode` and `/address` endpoints, now that devs.DC has the `/elections` endpoint, the direction we should really be heading in is that a ballot object from the [/postcode](https://developers.democracyclub.org.uk/api/v1/#postcode-search-2) or [/address](https://developers.democracyclub.org.uk/api/v1/#address-search-2) is a cut-down version of a ballot object from [/elections](https://developers.democracyclub.org.uk/api/v1/#elections-2), so I wouldn't want to randomly expose `organisation_type` as a top level ballot key when it isn't a top-level key on `/elections`. We should probably include (minimal?) `organization` / `division` keys in the `/postcode` and `/address` ballot objects.

Reply:
I was assuming we'd just use it for ordering and not present it to the front-end. Not sure there's any value in exposing it?

Reply:
In that case, it doesn't really matter